tdrs-backend/reports/zap.conf

```diff
@@ -121,4 +121,4 @@
 100000	WARN	(A Client Error response code was returned by the server)
 100001	WARN	(Unexpected Content-Type was returned)
 100001	OUTOFSCOPE	http://web:8080/v1/login
-100001	OUTOFSCOPE	https://tdp-backend-staging.app.cloud.gov/v1/login
+100001	OUTOFSCOPE	https://tdp-backend-staging.app.cloud.gov/v1/login
```

Comment:
no change here. temporarily add non-TDP URLs to confirm that `outofscope` doesnt help with exclusions related to 100001.